### PR TITLE
pass orga in param #1

### PR DESCRIPTION
**Pull request by ripoul**
Allow user to pass the organisation in paramamter to fetch datas.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
   const githubId = process.env.GITHUB_ID
   const githubToken = process.env.GITHUB_OAUTH
-  const githubOrganization = process.env.GITHUB_ORGA
+  const githubOrganization = process.argv[2]
```
Comment (frinyvonnick) [inline index.js]: Maybe you could use the environment variable if the param is not present ?

**Comment by frinyvonnick** [comment]
Nice work @ripoul 👍

File: index.js (hunk 1 above)
Comment (ripoul) [reply index.js]: For the orga and stats files?

Comment (frinyvonnick) [reply index.js]: Sure 😄

**Comment by jlandure** [comment]
Nice work! 👏

File: README.md [hunk 2]
```diff
@@ -28,11 +27,13 @@ You have to create a token with the `read:org` access.
...
 ```
 
 The following command allows you to get insights from the information
 
 ```
-yarn stats
+yarn stats <organization_name>
 ```
+
+You can add the organisazation name in the .env file. In this case use the key `GITHUB_ORGA`. If the two chose are set the env variable will be take.
```
Comment (frinyvonnick) [inline README.md]: Two small changes and we are done 👍 
`If the two chose` => `If both choices` 
`take` => `taken`

**Pull request merged**